auditbeat/module/file_integrity/event.go

```diff
@@ -400,6 +401,10 @@ func buildMetricbeatEvent(e *Event, existedBefore bool) mb.Event {
 		out.MetricSetFields.Put("process", process)
 	}
 
+	if e.ContainerID != "" {
+		out.MetricSetFields.Put("container.id", e.ContainerID)
```

Comment:
Should this be something like this:
```
container := mapstr.M{
    "id": e.ContainerID,
}

out.MetricSetFields.Put("container", container)
```

Or does `MetricSetFields.Put` handle dotted paths?

Reply:
so when you use `.Put(...)` the dot is separated automatically in a nested object as being done [here](https://github.com/elastic/beats/pull/38328/files#diff-908ccbec2a6448f00637da326cdfd893c01d540802728329ac3be463cd0585bbR419) but I can definitely see why it got your eye. If you want this changed no prob by me

Reply:
TIL ... thanks LGTM